R/vision-utils.R

```diff
@@ -0,0 +1,59 @@
+#' Normalizes a tensor to values between 0 and 1.
+#' 
+#' @param x tensor to normalize
+#'
+#' @export
+normalize <- function(x) {
```

Comment:
I think we should use a `prefix` here. Perhaps `transform_`?

Reply:
In Python, this is an inner function in `make_grid` (not a transform)... Perhaps keep it until we have transforms? Or should do  like in Python (inner function) and add another argument that says whether to normalize or not?

Reply:
+1 for using prefixes, `vision_` prefix maybe for all torch vision utils?

Reply:
Yeah, @dfalbel & I discussed this at length yesterday, but it seems like there is no good solution - `normalize` is like a transform, but unlike other transforms, not a class (otherwise something like `transform_min_max_scale` would have made sense...).

Now in the meantime I've been thinking, I'd rather just integrate this into `make_grid` for now... and if we wanted to do that and also use `vision_` like you suggest @javierluraschi , and then if we want to take into account that `vision` mainly has `datasets` and `models` and `transform`s, and (very few) `utils`, I think we could have things like

`vision_utils_make_grid`
`vision_transforms_bla`
`vision_models_bla` 

for example... This would be long though. What do you think?

Reply:
I would go with `vision_make_grid`. Since transforms and models are first class objects in vision they will probably be names like `transform_*` and `model_*`.

I also agree with `vision_utils_make_grid`. It's a large name, bt this function is not used many times in the script so that should be fine..

Reply:
> I would go with vision_make_grid. Since transforms and models are first class objects in vision they will probably be names like transform_* and model_*.

sounds perfect to me (all three do ;-)) 

Usability >> consistency for consistency's sake :-)